nixos/modules/installer/sd-card/sd-image-aarch64.nix

```diff
@@ -58,22 +58,23 @@
         # when attempting to show low-voltage or overtemperature warnings.
         avoid_warnings=1
       '';
-      in ''
-        (cd ${pkgs.raspberrypifw}/share/raspberrypi/boot && cp bootcode.bin fixup*.dat start*.elf $NIX_BUILD_TOP/firmware/)
+      in pkgs.runCommand "sdFirmware" {} ''
```

Comment:
Minor thing: I think it would be more clear to change the name argument for this derivation, maybe to `firmwarePartitionContents` as well or at least something that doesn't explicitly mention 'sd' anymore?